x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/470_connector_sync_job_list.yml

```diff
@@ -166,6 +166,52 @@ setup:
   - match: { count: 1 }
   - match: { results.0.id: $sync-job-one-id }
 
+---
+"List Connector Sync Jobs - Get jobs with single job type":
```

Comment:
NIT: we could also add a test that checks for illegal job type(s) passed to the endpoint, to make sure that it yields 400 response and doesn't accidentally cause some unhandled exceptions. Up to you if you want to add it :)

```
  - do:
      catch: "bad_request"
      connector_sync_job.list:
          connector_id: connector-one
          job_type: illegal_type,incremental

 ```

Reply:
Added for both `status` and `job_type`